### PR TITLE
Update to Spine v0.6.0

### PR DESCRIPTION
**Pull request by dmdashenkov**

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

**Comment by dmdashenkov** [comment]
@dmytro-kashcheiev PTAL
_Note:_ this is required by `gae-java` to execute sample `Application` successfully.

File: server-examples/src/main/java/org/spine3/examples/util/Users.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package org.spine3.examples.util;
+
+import org.spine3.users.UserId;
+
+/**
+ * Utility class for working with users.
+ *
+ * @author Dmytro Dashenkov
+ */
+public class Users {
```
Comment (dmytro-kashcheiev) [inline server-examples/src/main/java/org/spine3/examples/util/Users.java]: I think we need `@SuppressWarnings("UtilityClass")`  here.

File: server-examples/src/main/java/org/spine3/examples/eventstore/SampleData.java [hunk 2]
```diff
@@ -49,6 +49,7 @@
             newUserId("a.reid@spine3.org")
     );
 
+
```
Comment (dmytro-kashcheiev) [inline server-examples/src/main/java/org/spine3/examples/eventstore/SampleData.java]: Extra line.

**Comment by dmdashenkov** [comment]
@armiol We need to publish this to Maven

**Pull request merged**